plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -123,11 +123,6 @@ private function processMarkdown( $p_string ){
 
 		$t_string = MantisMarkdown::convert_text( $t_string );
```

Comment:
That make sense, we can move this logic also  `$t_string = str_replace( "&gt;", ">", $t_string );` to the class Extension.